src/Context.js

```diff
@@ -125,7 +125,8 @@ export default class Context {
       this.attachment(filename)
     }
 
-    yield send(this, path, options)
+    // NOTE: waiting for https://github.com/koajs/send/issues/38
+    yield send(this, path, options).call(this)
```

Comment:
'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.